plugins/catalog/src/components/AboutCard/IconLinkVertical/IconLinkVertical.tsx

```diff
@@ -41,9 +49,20 @@ const useIconStyles = makeStyles({
 export function IconLinkVertical({
   icon = <LinkIcon />,
   href = '#',
+  disabled = false,
   ...props
 }: IconLinkVerticalProps) {
   const classes = useIconStyles();
+
+  if (disabled) {
+    return (
+      <Link className={classes.disabled} underline="none" {...props}>
+        {icon}
+        <span className={classes.label}>{props.label}</span>
+      </Link>
+    );
+  }
+
   return (
     <Link className={classes.link} href={href} {...props}>
```

Comment:
no need for the repetition; add ONLY `color: gray` to the `disabled` class, and then something like

```ts
import classnames from 'classnames';
// ...
<Link className={classnames(classes.link, { [classes.disabled]: disabled })} underline="none" {...props}>
```

Reply:
@freben done!